### DISCUSSION TITLE
GitBook is now listed on resource.fyi!

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#143] [Announcements]
We've just launched GitBook on resource.fyi, a site for resources, products, and tools curated for helping you work better. 

<img width="1624" alt="Screenshot 2023-03-13 at 18 46 22" src="https://user-images.githubusercontent.com/42930383/224679850-5127337c-9ab2-4717-a3d3-2d986010fe71.png">

If you'd like, you feel free to upvote us here! 😄 https://resource.fyi/products/gitbook

**Comment by ghost** [comment]
Nice! Voted
